ERCS/erc-7798.md

```diff
@@ -0,0 +1,200 @@
+---
+eip: 7798
+title: Tap to Pay
+description: Initializing contactless payment transactions from EVM wallets
+author: Amhed Herrera (@amhed), Justin Lee (@JustinDLee), Arjun Dureja (@arjun-dureja)
+discussions-to: https://ethereum-magicians.org/t/erc-7798-contactless-payment/21501
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-25
+requires: 20, 681, 712
+---
+
+## Abstract
+This ERC defines a standard for contactless payment transactions that can allow for interoperable customer to merchant onchain payments, regardless of the wallets the customer and merchant are using.
+
+## Motivation
+Currently there is no standard mechanism in crypto to do contactless payment transactions via NFC. This ERC defines a standardized way of exchanging payment information between merchants and customers, or peer-to-peer individuals, so that efficient checkout can occur.
+
+## Specification
+
+Comprised of three parts:
+- A new Ethereum Provider JavaScript API method called `requestContactlessPayment`
+- An agreement on the payload for data exchange between the parties
+- An optional mechanism for relaying large JSON payloads using a backend relayer
+
+
+```mermaid
```

Comment:
Our renderer doesn't support mermaid. Please replace it with an image in your assets directory (preferably an SVG.)